packages/flat-components/src/components/SaveAnnotationModal/index.tsx

```diff
@@ -0,0 +1,128 @@
+import "./style.less";
+
+import pLimit from "p-limit";
+import classNames from "classnames";
+import React, { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Modal, message } from "antd";
+
+function download(
+    canvas: HTMLCanvasElement,
+    filename = "annotation.png",
+    failText = "Save image failed",
+): void {
+    try {
+        const a = document.createElement("a");
+        a.download = filename;
+        a.href = canvas.toDataURL();
+        a.style.display = "none";
+        document.body.appendChild(a);
+        a.click();
+        document.body.removeChild(a);
```

Comment:
```suggestion
        a.click();
```

No need `appendChild`